cardano-api-test/test/Test/Cardano/Api/TxBody.hs

```diff
@@ -0,0 +1,263 @@
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Test.Cardano.Api.TxBody (tests) where
+
+import           Cardano.Prelude
+
+import           Hedgehog (forAll, property, tripping)
+import           Test.Tasty (TestTree, testGroup)
+import           Test.Tasty.Hedgehog (testProperty)
+import           Test.Tasty.TH (testGroupGenerator)
+
+import           Cardano.Api
+
+import           Gen.Cardano.Api.Typed (genTxBody, genTxBodyContent)
+
+
+test_roundtrip_TxBody_make_get :: [TestTree]
+test_roundtrip_TxBody_make_get =
```

Comment:
I think this is sufficiently interesting and tricky to justify a decent comment on how this round trip test works. What are we going from and to? Why is there normalisation involved, and what normalisation are we doing? What is `review` about etc. Perhaps draw out the diagram of the conversions we're checking are equal.

It might be good to put the other property `test_roundtrip_TxBody_get_make` immediately after and document that too, to show the other direction we're checking. Seeing both together will make it clearer what we're checking overall. And presumably that one will be the same diagram, but checking different paths through the diagram, right?